bld/namelist_files/namelist_defaults_cam.xml

```diff
@@ -2014,13 +2014,17 @@
 <soil_erod_file phys="cam4"  >atm/cam/dst/dst_source2x2tuned-cam4-06132012.nc</soil_erod_file>
 <soil_erod_file phys="cam4" hgrid="0.9x1.25">atm/cam/dst/dst_source1x1tuned-cam4-06202012.nc</soil_erod_file>
 
-<!-- nitrogen deposition to surface models -->
+<!-- nitrogen deposition to surface components -->
+<stream_ndep_varlist>NDEP_NHX_month:NDEP_NOy_month</stream_ndep_varlist>
 <stream_ndep_mesh_filename>share/meshes/fv0.9x1.25_141008_polemod_ESMFmesh.nc</stream_ndep_mesh_filename>
 <stream_ndep_data_filename>lnd/clm2/ndepdata/fndep_clm_hist_b.e21.BWHIST.f09_g17.CMIP6-historical-WACCM.ensmean_1849-2015_monthly_0.9x1.25_c180926.nc</stream_ndep_data_filename>
-
-<stream_ndep_mesh_filename sim_year="1850">share/meshes/fv0.9x1.25_141008_polemod_ESMFmesh.nc</stream_ndep_mesh_filename>
 <stream_ndep_data_filename sim_year="1850">lnd/clm2/ndepdata/fndep_clm_WACCM6_CMIP6piControl001_y21-50avg_1850monthly_0.95x1.25_c180802.nc</stream_ndep_data_filename>
 
+<stream_ndep_varlist       phys="cam7">drynhx:wetnhx:drynoy:wetnoy</stream_ndep_varlist>
```

Comment:
We normally keep all the defaults for the same variable together. Of course, this is enforced with the new namelist syntax. Please move this to after line 2018.